tgui/packages/tgui/interfaces/NtosAtmos.js

```diff
@@ -0,0 +1,31 @@
+import { useBackend } from '../backend';
+import { Box, Button, LabeledList, Section } from '../components';
+
+export const NtosAtmos = props => {
+  const { act, data } = useBackend(props);
+  const air = data.AirData || [];
+  return (
+    <Section
+      title="Atmospheric Readings"
+      textAlign="center">
+      <Box>
+        Temperature: {data.AirTemp} Celcius
+      </Box>
+      <Box>
+        Pressure: {data.AirPressure} kPa
```

Comment:
what happens if T is null in the ui_data proc and AirPressure + AirTemp are undefined

Reply:
> what happens if T is null
This means, AirTemp and AirPressure won't be added to ui_data, and due to the way I've implemented the data receiving loop in tgui, it will keep showing last known values of AirTemp and AirPressure.
> ...and AirPressure + AirTemp are undefined
If they are truly undefined, they just won't be printed.